examples/js/postprocessing/EffectComposer.js

```diff
@@ -75,6 +77,22 @@ Object.assign( THREE.EffectComposer.prototype, {
 
 	},
 
+	isLastEnabledPass: function ( passIndex ) {
+
+		for ( i = passIndex + 1; i < this.passes.length; ++i ) {
```

Comment:
```js
for ( var i = passIndex + 1; i < this.passes.length; i ++ ) {
```

Reply:
Huh, that was a silly mistake. Amended the commit now with "var".